### ISSUE TITLE
Multi Material Printer - find proper location and rearrange electronics room

### ISSUE THREAD

**Issue by NitramLegov** [comakingspace/do-something#32]
The Multi Material printer works and got documented. As of this, we should now find a proper location for it, which might lead to a rearrangement of the electronics room.
The current proposal by Ciaran is to move the table the Multi Material printer is currently standing on to the other side. Then the printer could sit next to the hole in the wall (for adding a fan later, once we have a proper case).
The table with the power supplies, measurement tools, etc. could then go where the Multi Material printer currently stands.
Let´s discuss this on Friday evening.

- [x] Discuss proper location
- [x] rearrange electronics room
- [x] Update 3D Printer Farm and include P6 as being available

**Comment by emka** [comment]
Thanks for the documentation! I removed the tutors (copied from MK2S) since we have no multi-material experience yet. The info box links to a status list where the multi-material printer is still missing. Was it labeled P6?

For the location I do not have a strong opinion, however the table with power supplies etc is quite heavy and if I remember correctly we tried the other positions and it did not fit (measuring is easier than moving!).

When adding a fan for ABS printing I would rather use a regular model than the multi material.
All printers could get a case and a filter even without blowing the air outside, the main limitation is the material cost (no funds for that yet).
